README.md

```diff
@@ -166,6 +166,39 @@ this will map to the following `vrrp_instance` section
     }
 ```
 
+## Advanced Users Only: Override Keepalived Configuration Template
+
+Create a ConfigMap with the full contents of this configuration template file:
+https://github.com/redhat-cop/keepalived-operator/blob/29402297ca8a252be77904c3f3c29b08e50a3b85/config/templates/keepalived-template.yaml#L166
```

Comment:
does this have to be in blob format? Can we do just a link to the file on master?

Reply:
Yeah absolutely